docs/pages/docs/getting-started/app-router-server-components.mdx

```diff
@@ -0,0 +1,260 @@
+import Callout from 'components/Callout';
+import Steps from 'components/Steps';
+
+# Next.js 13: Internationalization (i18n) in Server Components
+
+Next.js 13 introduces support for [React Server Components](https://nextjs.org/docs/getting-started/react-essentials) and unlocks [many benefits](/docs/environments/server-client-components) when handling internationalization entirely on the server side. `next-intl` is adopting the new capabilities and is currently offering a beta version to early adopters, who are already building apps with the `app` directory.
+
+<Callout type="warning">
+  Support for React Server Components is currently in beta. Please use it at
+  your own risk, knowing that you may have to migrate upon a stable release.
+</Callout>
+
+## Current beta version
+
+```
+npm install next-intl@3.0.0-beta.5
```

Comment:
Are you going to release it as 3.0.0 and give access to the server components API, or will we still have to use the latest beta to use this API ?

Reply:
It will still be in beta for now, I'm waiting for full static rendering support to be able to call it stable! 🙂 See [the details here](https://next-intl-docs-git-docs-app-router-first-next-intl.vercel.app/docs/getting-started/app-router-server-components#static-rendering).